sdk/cognitiveservices/InkRecognizer/azure-ai-inkrecognizer/src/WPF/Azure.AI.WPF.Stroke/Azure.AI.InkRecognizer.WPF.Stroke.csproj

```diff
@@ -0,0 +1,120 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
To build successfully in our engineering system this project needs to be a new style SDK project (i.e. `<Project Sdk="Microsoft.NET.Sdk">`) and the libraries generally target netstandard2.0.